### PR TITLE
Edit test checks for banner formatting

### PR DESCRIPTION
**Pull request by cyarbrough76**
#### Description:

- _Edit tests for banner_etc_issue and banner_etc_motd for banner formatting_

#### Rationale:

- _tests cases were incomplete or banner formatting was failing scan_

### CODE REVIEW

**Comment by matejak** [comment]
For a bit of context - original banners were introduced in https://github.com/ComplianceAsCode/content/pull/2404, i.e. more than 2 years ago. Anyway, what was wrong with them?

**Comment by cyarbrough76** [comment]
> 
> 
> For a bit of context - original banners were introduced in #2404, i.e. more than 2 years ago. Anyway, what was wrong with them?

I believe it was just spacing and line returns causing it to fail scans.

**Comment by redhatrises** [comment]
The banners currently comply with what DISA provided in their notice consent document as well as what DoD CIO distributed out to the various DoD departments including what is in the SRG. These changes appear to undo that, so an actual policy document would be beneficial here.

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_motd/tests/banner_etc_motd_ospp_usbcg_banner.pass.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```
Comment (matejak) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_motd/tests/banner_etc_motd_ospp_usbcg_banner.pass.sh:3]: Are you sure about the profile here? `xccdf_org.ssgproject.content_profile_ospp` is not STIG, except that our RHEL8 STIG draft is based on it.

**Comment by matejak** [comment]
Editing tests in this way is strange - it doesn't change the behavior of the content, and usually if we want to test for something else, we add tests, we don't delete old ones.
If the old behavior became undesirable, we should mark old scenarios as "expected failures", and also change the OVAL check.
Therefore, I propose to close this PR if this concern or @redhatrises concern aren't answered in a reasonable timeframe.

**Comment by redhatrises** [comment]
PR is stale. Closing

**Pull request closed without merging**